src/server/chromium/crNetworkManager.ts

```diff
@@ -152,23 +152,23 @@ export class CRNetworkManager {
       this._attemptedAuthentications.add(event.requestId);
     }
     const {username, password} = this._credentials || {username: undefined, password: undefined};
-    this._client._sendMayFail('Fetch.continueWithAuth', {
+    void this._client._sendMayFail('Fetch.continueWithAuth', {
       requestId: event.requestId,
       authChallengeResponse: { response, username, password },
     });
   }
 
   _onRequestPaused(workerFrame: frames.Frame | undefined, event: Protocol.Fetch.requestPausedPayload) {
     if (!this._userRequestInterceptionEnabled && this._protocolRequestInterceptionEnabled) {
-      this._client._sendMayFail('Fetch.continueRequest', {
+      void this._client._sendMayFail('Fetch.continueRequest', {
         requestId: event.requestId
       });
     }
     if (!event.networkId) {
       // Fetch without networkId means that request was not recongnized by inspector, and
       // it will never receive Network.requestWillBeSent. Most likely, this is an internal request
       // that we can safely fail.
-      this._client._sendMayFail('Fetch.failRequest', {
+      void this._client._sendMayFail('Fetch.failRequest', {
```

Comment:
Perhaps we should make `_sendMayFail` not return a promise?

Reply:
There are methods like `InterceptableRequest.continue` that wait on the returned promise.

Reply:
Ok, updated the code.